designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -216,21 +292,24 @@
 # Tighten transition constraint for clocks declared so far
 set_max_transition {0.1*time_scale} -clock_path [get_clock clk_jtag]
 set_max_transition {0.1*time_scale} -clock_path [get_clock clk_retimer]
-set_max_transition {0.1*time_scale} -clock_path [get_clock clk_prbsgen]
+set_max_transition {0.025*time_scale} -clock_path [get_clock clk_tx_pi_0]
+set_max_transition {0.025*time_scale} -clock_path [get_clock clk_tx_pi_1]
+set_max_transition {0.025*time_scale} -clock_path [get_clock clk_tx_pi_2]
+set_max_transition {0.025*time_scale} -clock_path [get_clock clk_tx_pi_3]
+set_max_transition {0.05*time_scale} -clock_path [get_clock clk_tx_hr_0]
+set_max_transition {0.05*time_scale} -clock_path [get_clock clk_tx_hr_1]
+set_max_transition {0.1*time_scale} -clock_path [get_clock clk_tx_qr_0]
+set_max_transition {0.1*time_scale} -clock_path [get_clock clk_tx_qr_1]
 
 # Set transition time for high-speed signals monitored from iacore and itx
-# transition time is 10% of a 4 GHz period.  Note that we have to
-# create a clock before the transition constraint is applied
-# (appears to be related to the fact that these are internal nets)
+# transition time is 10% of a 4 GHz period.
 
 foreach x [get_object_name $adbg_clk_pins] {{
-    create_clock -name clk_mon_net_$x -period {0.25*time_scale} [get_pins $x]
```

Comment:
No longer using ``create_clock`` for clock signals that are only for debugging purposes.